src/views/playground/Playgrounds/PlaygroundCommonStore.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+
+import { createStore } from '@/store'
+
+type TestStoreState = {
+  newCount: number
+  test: number
+}
+
+type TestStoreActions = {
+  increment: (by: number) => void
+  decrement: (by: number) => void
+}
+
+const useStore = createStore<TestStoreState, TestStoreActions>(
+  {
+    state: { newCount: 0, test: 1 },
+    actionsFactory: (set) => ({
```

Comment:
Why not just `actions`?

Reply:
Wanted to make it clear that this isn't like state (just an object with properties being actions), but rather a function that given a `(set, get)` will return an object